pegasus/test/test_hoc_routes.rb

```diff
@@ -105,6 +105,33 @@ class HocRoutesTest < Minitest::Test
       end
     end
 
+    it 'starts and ends given tutorial, tracking company and tutorial' do
```

Comment:
good catch. I ran it locally with the correct 'testcompany' and it passed and then ran it again with 'wrongtestcompany' and it failed. I can do a followup for next week with a name change though. Or should I do it now?